### ISSUE TITLE
Consider adding a human intervention

### ISSUE THREAD

**Issue by chen9965** [OthersideAI/self-operating-computer#140]
Consider adding a human intervention? Like AutoGPT, there are times when the program repeats an impossible task, and we can manually intervene.

**Comment by Yash-1511** [comment]
Hey @chen9965 

It will be a great plan to add human feedback or intervention during running program. i think i have seen this in AutoGPT as well as Agent GPT. 

look forward to this feature. what's your opinion on this @joshbickett ?

**Comment by joshbickett** [comment]
@Yash-1511 @chen9965, I think human intervention could help in some cases, but if it makes sense since this project's vision is to let it be a "self-operating computer". It may make sense to fork this project into something different to add this.

**Comment by retsamcam** [comment]
> Hey @chen9965
> 
> It will be a great plan to add human feedback or intervention during running program. i think i have seen this in AutoGPT as well as Agent GPT.
> 
> look forward to this feature. what's your opinion on this @joshbickett ?

Yeah it is like witn AppAgent. I reckon @joshbickett is right: should we just fork it and replicate bascially what AppAgent has done for android but on desktop? Using say, pyautogui and threading libraries @Yash-1511 ?
